### ISSUE TITLE
Panel: aligned to the right

### ISSUE THREAD

**Issue by warpdesign** [linuxmint/Cinnamon#7457]
```
 * Cinnamon version 3.3.6
 * Distribution - Mint 18.3
 * Graphics hardware: intel 3/4th integrated gpu
 * 64 bit
 ```

**Issue**
In order to have a taskbar appear on my second monitor, I added a new panel, and moved it onto the second monitor.

I then added a few applets, like `window list`. It works, but applets seem to align to the right on the panel (while on the other panel that was there by default, everything is aligned onto the right):

![image](https://user-images.githubusercontent.com/199648/38665019-9a271552-3e3b-11e8-8262-54a290ccd9e4.png)

How can I make it align to the left ?

**Comment by NikoKrause** [comment]
The panel is separated into 3 parts (left, middle, right).
Right click on panel, turn on `Panel Edit Mode`. You see those 3 parts highlighted (with the Mint-X theme it's red, green, gray). You can drag the applets to the position you want. When you're finished, turn off panel edit mode.

![Video](https://streamable.com/zmhjc)

**Comment by Uliss2do** [comment]
Yeah, DRAG the window list exactly ABOVE the "centre separator" & release their,
The best thing about cinnamon & linux is that you learn new tricks everyday!

**Comment by brain90** [comment]
> The panel is separated into 3 parts (left, middle, right).
> Right click on panel, turn on `Panel Edit Mode`. You see those 3 parts highlighted (with the Mint-X theme it's red, green, gray). You can drag the applets to the position you want. When you're finished, turn off panel edit mode.
> 
> [Video](https://streamable.com/zmhjc)

Holly molly. that red, green, gray area was very difficult to find.  
thanks!
